docs/resources/hss_quota.md

```diff
@@ -0,0 +1,108 @@
+---
+subcategory: "Host Security Service (HSS)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_hss_host_protection"
+description: |-
+  Manages an HSS quota resource within HuaweiCloud.
+---
+
+# huaweicloud_hss_quota
+
+Manages an HSS quota resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "quota_version" {}
+
+resource "huaweicloud_hss_quota" "test" {
+  version     = var.quota_version
+  period_unit = "month"
+  period      = 1
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region to which the HSS quota resource belongs.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `version` - (Required, String, ForceNew) Specifies protection quota version. Changing this parameter will create a
+  new resource.  
+  The valid values are as follows:
+  + **hss.version.basic**: Basic version.
+  + **hss.version.advanced**: Professional version.
+  + **hss.version.enterprise**: Enterprise version.
+  + **hss.version.premium**: Ultimate version.
+  + **hss.version.wtp**: Web page tamper prevention version.
+  + **hss.version.container.enterprise**: Container version.
+
+* `period_unit` - (Required, String, ForceNew) Specifies the charging period unit of the quota.
+  Valid values are **month** and **year**. Changing this parameter will create a new resource.
+
+* `period` - (Required, Int, ForceNew) Specifies the charging period of the quota. Changing this parameter will
+  create a new resource.  
+  If `period_unit` is set to **month**, the value ranges from `1` to `9`.  
+  If `period_unit` is set to **year**, the value ranges from `1` to `5`.
```

Comment:
Valid values for `year` in console is 1, 2, 3, 5.

Reply:
done.